registry/zookeeper/service_discovery_test.go

```diff
@@ -64,6 +65,8 @@ func TestNewZookeeperServiceDiscovery(t *testing.T) {
 	name := "zookeeper1"
 	_, err := newZookeeperServiceDiscovery(name)
 
+	time.Sleep(time.Second)
```

Comment:
Why add sleep?

Reply:
Thanks for mentioned, It's just for test, Now I delete it.